guides/modulepath.txt

```diff
@@ -0,0 +1,25 @@
+TOPIC
+    modulepath
+
+DESCRIPTION
+    The modulepath is an ordered list of directories that Bolt loads modules
+    from. When Bolt runs a command, it automatically loads modules from the
+    modulepath.
+    
+    While Bolt has a default modulepath, you can also configure your own
+    modulepath, which can include directories within the project or directories
+    elsewhere on your system. Regardless of whether your project uses a default
+    or configured modulepath, Bolt automatically adds a couple directories to
```

Comment:
```suggestion
    or configured modulepath, Bolt automatically adds two directories to
```
It's just the two now right? If it's a small, but undefined number, I would maybe use different phrasing - For example "certain directories" / "specific directories" or forego the descriptor and just use "...adds directories...".

Reply:
_ish_. It adds `.modules` if you're using the module management workflow, and the project if you're using a named project. I'll be less specific.